wp-admin/includes/class-wp-plugin-dependencies.php

```diff
@@ -388,20 +388,24 @@ private function get_view_details_link( $plugin_file, $names ) {
 		foreach ( $dependencies as $dependency ) {
 			$plugin_data = $this->plugin_data[ $dependency ];
 			foreach ( $names_arr as $name ) {
-				if ( $name === $plugin_data['name'] ) {
-					if ( empty( $plugin_data['version'] ) ) {
-						$details_links[ $name ] = $name;
-					} else {
-						$details_links[ $name ] = sprintf(
-							"<a href='%s' class='thickbox open-plugin-details-modal' aria-label='%s' data-title='%s'>%s</a>",
-							network_admin_url( 'plugin-install.php?tab=plugin-information&plugin=' . $plugin_data['slug'] . '&TB_iframe=true&width=600&height=550' ),
-							/* translators: %s: Plugin name. */
-							sprintf( __( 'More information about %s' ), $name ),
-							$name,
-							$name
-						);
-					}
+				if ( $name !== $plugin_data['name'] ) {
+					continue;
+				}
+
+				if ( empty( $plugin_data['version'] ) ) {
+					$details_links[ $name ] = $name;
+					continue;
 				}
+
+				$url                    = network_admin_url( 'plugin-install.php?tab=plugin-information&plugin=' . $plugin_data['slug'] . '&TB_iframe=true&width=600&height=550' );
+				$name_attr              = esc_attr( $name );
```

Comment:
Curious why create a new variable that's only used once.

Reply:
That would be because I had previously added it in multiple areas by mistake. Fixed in latest version.